src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/dropdowncomponentobject/DropDownComponentObject.java

```diff
@@ -194,4 +194,8 @@ public void verifyMessageAboutNewPassword(String userName) {
 			true
 		);
 	}
+
+	public void verifyFBButtonVisible() {
```

Comment:
@jsutterfield I had the same instinct but the problem is that the name of the button is different depending on where I am looking for it -- so the one on the drop down has a different identifier than the one on the login page, which is different than the one on the login modal.  So I suppose I could define the function in WikiBasePageObject and then assign the variable in the specific page class?  I will try this.